immuni_analytics/helpers/safety_net.py

```diff
@@ -124,8 +178,14 @@ def _verify_signature(jws_token: str, leaf_certificate: crypto.x509) -> None:
     :param leaf_certificate: the leaf certificate extracted from the jws header.
     """
     public_key = crypto.dump_publickey(crypto.FILETYPE_PEM, leaf_certificate.get_pubkey())
-
-    jwt.decode(jws_token, public_key)
+    try:
+        jwt.decode(jws_token, public_key)
+    except DecodeError as exc:
+        _LOGGER.warning(
+            "Could not verify jws signature.",
+            extra=dict(error=str(exc), jws_token=jws_token, public_key=public_key),
+        )
+        raise SafetyNetVerificationError()
```

Comment:
Add raises for all method.

Reply:
Would public_key = crypto.dump_publickey(crypto.FILETYPE_PEM, leaf_certificate.get_pubkey()) raise a DecodeError?

Reply:
No, I meant `raises` sorry. In the docstring for all _methods_ in the file :D

Reply:
Are you the little picky one? 😝 Sure, I'll do ASAP. 🚀

Reply:
Done. ✅